PC/VS9.0/_hashlib.vcproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="Windows-1252"?>
 <VisualStudioProject
 	ProjectType="Visual C++"
-	Version="9,00"
+	Version="9.00"

```

Comment:
That's changed by VisualStudio itself in accordance with machine's region settings.  U.S. decimal point vs. European? decimal.